app/scripts/modules/core/src/pipeline/config/stages/shareService/shareServiceStage.ts

```diff
@@ -0,0 +1,18 @@
+import { module } from 'angular';
+
+import { Registry } from 'core/registry';
+import { ExecutionDetailsTasks } from '../common';
+import { ShareServiceExecutionDetails } from './ShareServiceExecutionDetails';
+
+export const SHARE_SERVICE_STAGE = 'spinnaker.core.pipeline.stage.shareService';
+
+module(SHARE_SERVICE_STAGE, []).config(() => {
```

Comment:
Look to `waitStage` for an example of why we don't need to wrap this in an angular component anymore.

Reply:
👍